Xamarin.Forms.Platform.UAP/CollectionView/SelectableItemsViewRenderer.cs

```diff
@@ -171,13 +171,15 @@ void UpdateFormsSelection()
 
 		void UpdateFormsSingleSelection()
 		{
-			var selectedItem = ListViewBase.SelectedItem is ItemTemplateContext itemPair 
-				? itemPair.Item 
+			var selectedItem = ListViewBase.SelectedItem is ItemTemplateContext itemPair
+				? itemPair.Item
 				: ListViewBase.SelectedItem;
-			
+
 			ItemsView.SelectionChanged -= FormsSelectionChanged;
 			ItemsView.SelectedItem = selectedItem;
-			ItemsView.SelectionChanged += FormsSelectionChanged;
+
+			if (ItemsView != null)
+				ItemsView.SelectionChanged += FormsSelectionChanged;
```

Comment:
Not seeing how adding the event handler will fix the NRE.

Reply:
The event handler existed, added the NRE check for `ItemsView`.